hexa/plugins/connector_dhis2/api.py

```diff
@@ -136,12 +136,14 @@ class IndicatorResult(Dhis2Result):
 
 
 class Dhis2Client:
+    # TODO: is this indirection really necessary? why not merge with models and simplify things?
```

Comment:
I would prefer to avoid having API / external interaction code in models. We have, however, a task in our roadmap to review our "sync patterns" and harmonize/refactor our connectors accordingly. This comment can be removed for now.